.ko.yaml

```diff
@@ -2,14 +2,14 @@ builds:
 - id: initContainer
   main: ./cmd/initContainer
   ldflags:
-  - "{{ .Env.LD_FLAGS }}"
```

Comment:
FYI in the latest `ko` release v0.12.0 released earlier this week, if the env is unset the build proceeds with just an empty value.

Reply:
I'm using 0.12 but experienced the same issue.

Reply:
Oh the change only affected `LDFLAGS`, without `_`: https://github.com/google/ko/pull/758/files 😢 

I'll make that better in a future release.

Reply:
We can rename to `LDFLAGS` ;)